src/main/java/org/broadinstitute/hellbender/utils/test/CommandLineProgramTestInterface.java

```diff
@@ -0,0 +1,94 @@
+package org.broadinstitute.hellbender.utils.test;
+
+import htsjdk.samtools.util.Log;
+import org.broadinstitute.hellbender.Main;
+import org.broadinstitute.hellbender.cmdline.StandardArgumentDefinitions;
+import org.broadinstitute.hellbender.utils.logging.BunnyLog;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+/**
+ * Utility interface for CommandLine Program testing.
+ */
+public interface CommandLineProgramTestInterface {
```

Comment:
Whenever you have an idea for this, just let me know. The alternative that you suggested sounds better than this, so I changed it in the next commit.